frontend/src/app/_components/BugReportModal/BugReportModal.tsx

```
@@ -0,0 +1,359 @@
+
+  const removeAttachment = useCallback((index: number) => {
+    setAttachments((prev) => {
+      const toRemove = prev[index];
+      URL.revokeObjectURL(toRemove.preview);
+      return prev.filter((_, i) => i !== index);
+    });
+  }, []);
+
+  const removeAllAttachments = useCallback(() => {
+    setAttachments((prev) => {
+      prev.forEach((a) => URL.revokeObjectURL(a.preview));
```

Biome (2.4.4)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)